src/main/java/baseball/GamePlayer.java

```diff
@@ -0,0 +1,36 @@
+package baseball;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import static baseball.Number.*;
+
+public class GamePlayer {
+
+    public static void play(Scanner Input){
+        Number random = new Number();
+        Number player = new Number();
+        Integer temp;
```

Comment:
사용자 입력을 받아서 그 상태를 가지고 있는 변수가 temp라는 이름으로 그 의도를 드러내지 못하고 있습니다.

클린코드라는 책에서는 한 챕터를 의미 있는 이름으로 할당할 만큼 의도가 분명한 이름의 중요성은 아무리 강조해도 지나치지 않은 것 같습니다! 코딩할 때 힘들더라도 의미 있는 이름을 지어주려고 노력해보세요. 😃

```suggestion
        Integer userInputNumber;
```